guides/micronaut-rabbitmq-rpc/bookrecommendation/java/src/main/java/example/micronaut/BookRecommendation.java

```diff
@@ -1,16 +1,16 @@
 package example.micronaut;
 
+import io.micronaut.core.annotation.Creator;
 import io.micronaut.core.annotation.Introspected;
 
 import java.util.Objects;
 
 @Introspected
 public class BookRecommendation {
-    private String name;
 
-    public BookRecommendation() {
-    }
+    private final String name;
 
+    @Creator
```

Comment:
Is `@Creator` necessary? If you want to include it, we should educate the user about it with a callout.